portal-web/test/functional/com/liferay/portalweb/tests/enduser/wcm/exportimport/ExportImport.testcase

```diff
@@ -305,6 +305,107 @@
 		</execute>
 	</command>
 
+	<command name="ExportImportDMRootFolder" priority="5">
```

Comment:
@anthony-chu, thanks for finding that.  It's safe to leave out of master (please remove if I added it).  I'll check with Maria if either of those integration tests were backported, and we may be able to streamline even more coverage on ee-7.0.x.